lib/wallaby/driver.ex

```diff
@@ -256,6 +256,16 @@ defmodule Wallaby.Driver do
     resp["value"]
   end
 
+  @doc """
+  Retrives the current page source from session
+  """
+  def source(session) do
```

Comment:
Lets change this to `page_source` since it can only ever be used to get the source of the page.